certificate-renewal/Certificate Renewal JSON contract.md

```diff
@@ -0,0 +1,169 @@
+# CD Renew JSON contract
+
+## Initial Request (PRELOADDATA):
+```json
+{
+  "rgState": "PRELOADDATA",
+  "powerOnFileName": "BANNO.CD.RENEW.V1.POW",
+  "userChrList":[
+    {"id": 1, "value": "0123456789S0123"},
+    {"id": 2, "value": ""},
+    {"id": 3, "value": ""},
+    {"id": 4, "value": ""},
+    {"id": 5, "value": ""}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+ - userCharList[1]: 10-digit account number, 'S' and 4-digit share ID
+	 - Represents the target certificate or club share
+
+
+## Successful Response:
+```json
+{
+  "results": {
+    "currentState": {
+      "name": "My Wonderful CD",
+      "maturityDate": "01/01/21",
+      "minimumBalance": 12345.67,
+      "currentStatus": "Renew",
+      "transferSLId": "1234567890S0010",
+      "transferAmount": "12345.67"
+    },
+    "options": [{
+        "name": "Increase certificate balance",
+        "id": 1
```

Comment:
Can id be changed back to value? It's easier and less code for me with the dropdown menu...

Reply:
I could have sworn that JR asked me to change them to ID... but now I can't see his comment.... so I guess I'll change them back?

Reply:
Yes he did but then we talked and he's okay with it being value - it works better for some of the code I'm writing